src/realm/table.hpp

```diff
@@ -518,6 +524,10 @@ class Table {
         return Query(m_own_ref, list);
     }
 
+    Query query(const std::string& query_string, const std::vector<Mixed>& arguments = {}) const;
+    Query query(const std::string& query_string, query_builder::Arguments& arguments,
+                const parser::KeyPathMapping&) const;
```

Comment:
Maybe you want a way to get a `DescriptorOrdering` as well? This can come later.

Currently, any sort/limit/distinct clauses will be ignored.